### PR TITLE
update schedule content

### PR DESCRIPTION
**Pull request by ellenli030**
content update for schedule

image

### CODE REVIEW

**Review by gordonpn — APPROVED** [review]
I think it looks a bit squished on mobile with the added content, but it's fine to me. I want to know what @sttic thinks!

Looks great to me!

**Comment by gordonpn** [comment]
Also, an idea: moving the speaker's name from the title into another line, below in the description?

File: components/ScheduleTable.tsx [hunk 1]
```diff
@@ -85,7 +91,8 @@ export default function ScheduleTable({
                 </Flex>
               </Td>
               <Td>
-                <Text>{event.title}</Text>
+                <Text fontSize="xl">{event.title}</Text>
```
Comment (sttic) [inline components/ScheduleTable.tsx:94]: > I think it looks a bit squished on mobile

I think so too @gordonpn

Maybe try with small or medium font size with bold font weight, and some margin on the bottom if a description is provided!

**Comment by ellenli030** [comment]
image

lmk if you guys are okay with this

**Review by sttic — APPROVED** [review]
Loods good!

**Pull request merged**